Extensions/timestamps.js

```diff
@@ -172,7 +183,7 @@ XKit.extensions.timestamps = new Object({
 
 			if (XKit.extensions.timestamps.in_search && !$("#search_posts").hasClass("posts_view_list")) {
 				var in_search_html = '<div class="xkit_timestamp_' + post_id + ' xtimestamp-in-search xtimestamp_loading">&nbsp;</div>';
-				post.find(".post_controls_top").prepend(in_search_html);
+				post.find(".post-info-tumblelogs").prepend(in_search_html);
```

Comment:
The added text is absolutely positioned so it doesn't change the size of the post card